docs/markdown/Using-wraptool.md

```diff
@@ -96,3 +96,16 @@ but available in WrapDB will automatically be downloaded.
 
 Automatic fetch of WrapDB subprojects can be disabled by removing the file
 `subprojects/wrapdb.json`, or by using `--wrap-mode=nodownload`.
+
+## Self-hosted Wrap database
+
+If for whatever reason you want to use self-hosted or proxied Wrap database *(since 1.X.X)* you may set mirror address to be used for the workspace:
```

Comment:
The grammar would require some commas in this sentence, however, I don't think that "for whatever reason" is an useful qualifier to have, and I would just leave it out. I also don't think the alternative Wrap database should be "self-hosted". It can be that someone simply wants to use a different server.

Reply:
Sure.